Screen Wake Lock API subedit (#31208)
mdn/content: files/en-us/web/api/wakelock/request/index.md

    - The document is not fully active.
    - The document's visibility state is `hidden`.
    - The {{Glossary("User Agent")}} could not acquire platform's wake lock.
      For example, this might happen if the device is low on battery.

## Examples

The following asynchronous function requests a {{domxref("WakeLockSentinel")}} object.

> The `request()` method is wrapped in a `try...catch` statement to account for if the browser refuses the request for any reason.

Reviewer: ```suggestion
The `request()` method is wrapped in a `try...catch` statement to handle cases where the browser refuses the request for any reason.
```

Reviewer: Original wording seemed a bit awkward.

requestWakeLock();
```

> Note that the screen wake lock may be revoked by the device after it has been granted.

Reviewer: ```suggestion
The screen wake lock may be revoked by the device after it has been granted.
```

Reviewer: Or put it in a note box?

Author: I like your suggestion. Note not really needed.
